plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2575,51 +2553,25 @@ read_only::get_required_keys_result read_only::get_required_keys( const get_requ
    return result;
 }
 
-template<typename Params, typename Results>
-void read_only::send_transient_transaction(const Params& params, next_function<Results> next, chain::transaction_metadata::trx_type trx_type) const {
-   try {
-      auto pretty_input = std::make_shared<packed_transaction>();
-      auto resolver = make_resolver(db, abi_serializer::create_yield_function( abi_serializer_max_time ));
-      try {
-         abi_serializer::from_variant(params.transaction, *pretty_input, resolver, abi_serializer::create_yield_function( abi_serializer_max_time ));
-      } EOS_RETHROW_EXCEPTIONS(chain::packed_transaction_type_exception, "Invalid packed transaction")
-
-      app().get_method<incoming::methods::transaction_async>()(pretty_input, true /* api_trx */, trx_type, true /* return_failure_trace */,
-          [this, next](const next_function_variant<transaction_trace_ptr>& result) -> void {
-             if (std::holds_alternative<fc::exception_ptr>(result)) {
-                   next(std::get<fc::exception_ptr>(result));
-              } else {
-                 auto trx_trace_ptr = std::get<transaction_trace_ptr>(result);
-
-                 try {
-                    fc::variant output;
-                    try {
-                        output = db.to_variant_with_abi( *trx_trace_ptr, abi_serializer::create_yield_function( abi_serializer_max_time ) );
-                    } catch( chain::abi_exception& ) {
-                        output = *trx_trace_ptr;
-                    }
-
-                    const chain::transaction_id_type& id = trx_trace_ptr->id;
-                    next(Results{id, output});
-                 } CATCH_AND_CALL(next);
-              }
-      });
-   } catch ( boost::interprocess::bad_alloc& ) {
-      chain_plugin::handle_db_exhaustion();
-   } catch ( const std::bad_alloc& ) {
-      chain_plugin::handle_bad_alloc();
-   } CATCH_AND_CALL(next);
-}
-
-void read_only::compute_transaction(const compute_transaction_params& params, next_function<compute_transaction_results> next) const {
-   return send_transient_transaction(params, next, transaction_metadata::trx_type::dry_run);
+void read_only::compute_transaction(const compute_transaction_params& params, next_function<compute_transaction_results> next) {
```

Comment:
`params` should be by value so move works.

Reply:
done